tests/test_files/test_integration/alanine_hat_naive/kimmdy.yml

```diff
@@ -2,7 +2,7 @@ dryrun: false
 name: 'alanine_hat_000'
 max_tasks: 100
 gromacs_alias: 'gmx'
-gmx_mdrun_flags: -maxh 24 -dlb yes -nt 8 -npme 0 -ntmpi 1
+gmx_mdrun_flags: -maxh 24 -dlb yes
```

Comment:
Was this because gromacs complains about the mpi flag when not compiled with mpi?

Reply:
Yes exactly. Also, I think gromacs has become better at figuring these values out.